plugins/module_utils/foreman_helper.py

```diff
@@ -1028,7 +1028,7 @@ def __init__(self, **kwargs):
         foreman_spec.update(kwargs.pop('foreman_spec', {}))
         super(ForemanDataStreamModule, self).__init__(foreman_spec=foreman_spec, **kwargs)
 
-    def datastream_run(self):
+    def run(self):
```

Comment:
I know lazy n stuff...

But should it not support the same interface as the old one?

Reply:
I don't see a benefit implementing this at this point, tbh :)

Reply:
Good for you, that the 'new' run does not accept parameters anyway.